### DISCUSSION TITLE
Question about explicit time integrator

### DISCUSSION THREAD

**Discussion by SomnusYu** [idaholab/moose#27925] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hi all,

I would like to know why all the explicit time integrators required `solve_type = LINEAR`. If the solve type = PJFNK, is it impossible to use explicit time integrators?

Thanks!

**Comment by GiudGiud** [comment]
Hello

That's not true for all explicit time integrators.
See this one:
https://mooseframework.inl.gov/source/timeintegrators/ActuallyExplicitEuler.html

For those that do require it, it's because the implementation does not support nonlinear solves at the moment. It s something that can be changed with enough development work 

Guillaume

**Reply by SomnusYu** [reply]
Thanks!
